app/views/groups/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="flex flex-col justify-center h-full">
+  <div class="bg-main flex justify-between text-white items-center h-12">
+    <%= button_to "<", destroy_user_session_path, method: :delete, class: 'text-xl mx-2'  %>
+    <h1 class="font-proxima-bold">GROUPS</h1>
+    <%= link_to ">", new_group_path, class: 'text-xl mx-2'  %>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Home%20page%20(categories%20page)), This page supposes to be called `Categories` page, Please change this header to be `Categories` instead of `Groups`.

Reply:
Thanks @Omar-Muhamad  i am changing to match requeriments